package.json

```diff
@@ -12,10 +13,19 @@
     "url": "https://github.com/netcentric/fe-build"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\""
+    "clean": "del ./test/dist",
```

Comment:
And where is actual test?

Reply:
Github action has
```
      - name: Build Library
        run: npm run build --if-present
      - name: Run Tests
        run: npm test --if-present
```

I would add `build:prod` as default `test`.

Reply:
I removed the "test" task because running `npm run clean && node ./cli/index.js --production` fails in CI. I need to investigate why it is failing.

Reply:
Done, and it works now... 🤔